tests/core/test_channels.py

```diff
@@ -810,6 +1000,67 @@ async def test_slackbot_send_attachment_with_text():
         }
 
 
+@pytest.mark.filterwarnings("ignore:unclosed.*:ResourceWarning")
+@pytest.mark.asyncio
+async def test_slackbot_send_attachment_with_text_threaded():
+    from rasa.core.channels.slack import SlackBot
+
+    with aioresponses() as mocked:
+        mocked.post(
+            "https://www.slack.com/api/chat.postMessage",
+            payload={"ok": True, "purpose": "Testing bots"},
+        )
+
+        bot = SlackBot("DummyToken", "General", ts="DummyThread")
+        attachment = {
```

Comment:
Same here: probably better to define as a constant, which can be used in both `test_slackbot_send_attachment_with_text` and this test.